challenges/arrays-callbacks.js

```diff
@@ -21,6 +21,11 @@ The zoos want to display both the scientific name and the animal name in front o
 
 */
 const displayNames = [];
+zooAnimals.forEach(function(currentValue){
+  displayNames.push(
+    `Name: ${currentValue.animal_name}, Scientific: ${currentValue.scientific_name}.`
```

Comment:
Remember to commit and push your code where we moved from 'Scientific: ' on to the next line!